index.html

```diff
@@ -0,0 +1,69 @@
+<html>
+
+
+
+<main id="main">
+  
+    <h1 id="title"> Bombay Bicycle Club</h1>
+    <link rel="stylesheet" href="style.css">
+    <p>4 men Indie ensamble hailing from London</p>
+    <figure id="img-div">
+      <img
+        id="image"
+        src="https://upload.wikimedia.org/wikipedia/commons/thumb/e/e3/BombayBicycleClub.jpg/1280px-BombayBicycleClub.jpg"
+        alt="Bombay Bicycle Club playing at the University of Nottingham"
+      />
```

Comment:
- [ ]  Please add images to your repository assets, avoiding linking directly from another site.